SR2024/2023-11-15-kit-usb-hub-issue.md

```diff
@@ -0,0 +1,18 @@
+---
+to: Student Robotics 2024
+subject: SR2024 Power board connectivity issues
+---
+
+Hello all,
+
+We've seen reports in Discord from a few teams about issues around board connectivity where a "No boards of this type found" error is thrown for the power board, whilst the wiring and connections are seemingly correct.
+
+We've been able to track this down to a bug in Raspberry Pi OS (which our custom kit OS is based on), where plugging the motor board into certain ports on the USB hub causes all boards to be undetectable. If you're interested in the intricate details, see [here](https://github.com/raspberrypi/linux/issues/3779#issuecomment-709481662) and [here](https://groups.google.com/g/linux.debian.bugs.dist/c/5jI9dDZgfUU).
```

Comment:
For accessibly we should avoid using "here" as link text and instead prefer to have something which describes the target of the link. Perhaps:
```suggestion
We've been able to track this down to a bug in Raspberry Pi OS (which our custom kit OS is based on), where plugging the motor board into certain ports on the USB hub causes all boards to be undetectable. If you're interested in the intricate details, see [the issue on the Raspberry Pi OS issue tracker](https://github.com/raspberrypi/linux/issues/3779#issuecomment-709481662) and [discussion on the Debian Linux mailing list](https://groups.google.com/g/linux.debian.bugs.dist/c/5jI9dDZgfUU).
```